nativelib/src/main/scala/scala/scalanative/runtime/SocketHelpers.scala

```diff
@@ -0,0 +1,173 @@
+package scala.scalanative.runtime
+
+import scalanative.native._
+import scalanative.posix.{netdb, netdbOps}, netdb._, netdbOps._
+import scalanative.posix.arpa.inet._
+import scalanative.posix.sys.socketOps._
+import scalanative.posix.sys.socket._
+import scalanative.posix.sys.select._
+import scalanative.posix.unistd.close
+import scalanative.posix.sys.selectOps._
+import scalanative.posix.netinet.{in, inOps}, in._, inOps._
+
+object SocketHelpers {
+
+  def isReachableByEcho(ip: String, timeout: Int): Boolean = {
```

Comment:
Yes `scala.scalanative.runtime` is public.

Maybe move this object to the `java.net` namespace in the `nativelib`, make it `private[net]`, add the parameter and a test for it. I don't know what @densh will think of that, but I think it's important to test this functionality as I can easily be broken by a refactoring.